modules/tensor_mechanics/src/userobjects/TensorMechanicsPlasticTensileMulti.C

```diff
@@ -498,10 +498,10 @@ TensorMechanicsPlasticTensileMulti::returnPlane(const std::vector<Real> & eigval
 }
 
 bool
-TensorMechanicsPlasticTensileMulti::KuhnTuckerOK(const RankTwoTensor & returned_diagonal_stress, const std::vector<Real> & dpm, const Real & str) const
+TensorMechanicsPlasticTensileMulti::KuhnTuckerOK(const RankTwoTensor & returned_diagonal_stress, const std::vector<Real> & dpm, const Real & str, const Real & ep_plastic_tolerance) const
```

Comment:
Uncommon function arguments for KuhnTuckerOK:

This is for performance - evaluating the yield function (the std::vector thing) is expensive and not actually needed right here.